Toolchain/Patches/vcpkg/force-cmake-policy-min-3_5.patch

```diff
@@ -0,0 +1,25 @@
+From 313f400e7b5dd842b36de1de8ae1c34a42197939 Mon Sep 17 00:00:00 2001
+From: Nicole Patricia Mazzuca <nicole@strega-nil.co>
+Date: Mon, 31 Mar 2025 15:35:56 +0200
+Subject: [PATCH] [scripts/ports.cmake] set CMAKE_POLICY_VERSION_MINIMUM
+
+CMake 4 broke compatiblity with CMake pre-3.5, so we need to set this variable
+to avoid ports which have an earlier cmake_minimum_required(VERSION) breaking.
+---
+ scripts/ports.cmake | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/scripts/ports.cmake b/scripts/ports.cmake
+index 0ff66379e16d00..d2a071fc8bd4c2 100644
+--- a/scripts/ports.cmake
++++ b/scripts/ports.cmake
+@@ -3,6 +3,9 @@ if(POLICY CMP0174)
+     # Use CMake 3.31 behavior for cmake_parse_arguments(PARSE_ARGV)
+     cmake_policy(SET CMP0174 NEW)
+ endif()
++if(CMAKE_VERSION VERSION_GREATER_EQUAL "4.0")
++    set(ENV{CMAKE_POLICY_VERSION_MINIMUM} 3.5)
```

Comment:
Is this potentially something we could set via Meta/CMake/vcpkg/base-triplets/base.cmake or one of the file it includes?

Reply:
possibly.. given that https://github.com/microsoft/vcpkg/pull/44712 seems to be on the fast track, I might just wait until microsoft merges that (presumably it will be this evening sometime?) and drop the patch here.

Reply:
.... or not. going with a "base.cmake" version of the patch (actually putting it in the build-vcpkg-variables.cmake file)